### DISCUSSION TITLE
nft-geo-filter with fail2ban

### DISCUSSION THREAD

**Discussion by frankofno** [rpthms/nft-geo-filter#18] [Q&A]
Your script is working really well. Thanks for sharing that project. I was wandering, if I start using fail2ban, could your script interfere with fail2ban?

**Comment by rpthms** [comment]
Well, since nft-geo-filter uses a completely different table for it's rules, it shouldn't interfere with fail2ban's rules. But you do have to watch out for the parsing order of rules. Since nft-geo-filter uses a very high priority for its chains, it will probably come into affect earlier than the fail2ban rules (Not sure what priority fail2ban uses, I've never actually used it). So, you have to ensure that your intended traffic passes through nft-geo-filter so that it can reach fail2ban.
